### PR TITLE
Adding geo URI

### PR DESCRIPTION
**Pull request by david-dick**
Can we add the URI::geo type?

### CODE REVIEW

**Comment by david-dick** [comment]
Happy to assign you MAINT as before

File: lib/URI/geo.pm [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+
+__END__
+
+=head1 NAME
+
+URI::geo - The geo URI scheme.
+
+=head1 VERSION
+
+This document describes URI::geo version 5.26
```
Comment (oalders) [inline lib/URI/geo.pm:241]: I think we can remove this as Dist::Zilla should handle it.

**Review by oalders — APPROVED** [review]
Thanks, @david-dick! I guess you can transfer perms after release?

**Comment by david-dick** [comment]
okay, i've transferred ownership and removed the extra lines.  Should be good to go.

**Comment by oalders** [comment]
Thanks, @david-dick!

**Pull request merged**